conf/defaults.ini

```diff
@@ -1339,6 +1339,8 @@ plugin_catalog_url = https://grafana.com/grafana/plugins/
 plugin_catalog_hidden_plugins =
 # Log all backend requests for core and external plugins.
 log_backend_requests = false
+# Force download of public key for verifying plugin signature on startup.
+plugin_force_public_key_download = false
```

Comment:
Nit. Seems like we already use `force` in other places, but it's a rather "strong" word. How about `download_public_key_at_startup` or something similar? I left out `plugin_` since already in the context of `[plugins]`.

Reply:
I think the `force` part is needed here. If we have `download_public_key_at_startup = false` that could mean that we are not downloading the key but we do if it's empty or stale. My plan in the future is to remove the feature flag and then add a config field like `skip_public_key_download` which actually controls if the key is downloaded or not.

Reply:
> that could mean that we are not downloading the key but we do if it's empty or stale. 

Yeah that's true. But we would still have a description of the configuration where we can explain this.

Maybe `enforce` rather than `force` then? :shrug: